packages/web-runtime/src/pages/resolvePublicLink.vue

```diff
@@ -74,35 +74,36 @@
 <script type="ts">
```

Comment:
Could you fix this to `<script lang="ts">`?

Reply:
Done. Also grepped the codebase, there is no other occurence of `script type=`